civihr_employee_portal/js/ta-documents-app.js

```diff
@@ -7,11 +7,13 @@
       $urlRouterProvider.otherwise('/dashboard');
     })
     .controller('ModalController', ['$scope', '$rootScope', '$window', '$rootElement', '$log', '$uibModal',
-      'DocumentService', 'FileService', 'config', 'settings',
-      function($scope, $rootScope, $window, $rootElement, $log, $modal, DocumentService, FileService, config, settings) {
+      'DocumentService', 'FileService', 'config', 'settings', 'HR_settings',
+      function($scope, $rootScope, $window, $rootElement, $log, $modal, DocumentService, FileService, config,
+        settings, HR_settings) {
         var vm = {};
         var isContactsCached = {};
 
+        HR_settings.DATE_FORMAT = "dd/MM/yyyy";
```

Comment:
> It seems a bit weird to change the settings inside of a controller. It kind of kills the purpose of having a "global" settings var, right? Do we really need to change it in here?

Here the `HR_settings` factory comes form the `org.civicrm.reqangular` (civihr_employee_portal/js/ta-documents-app.js) extension. It is set to `null` by default ad following.
```js
define([
    'common/modules/services'
], function (module) {
    'use strict';

    module.factory('HR_settings', function () {
        return {
            DATE_FORMAT: null
        };
    });
});
```
So, I prefer to set the date format here in controller.